edge-functions/streaming-response/middleware.ts

```diff
@@ -0,0 +1,11 @@
+import { NextResponse, NextRequest } from 'next/server'
+
+export const config = {
+  matcher: '/',
+}
+
+export function middleware(req: NextRequest) {
+  const url = req.nextUrl
+  url.pathname = '/api/edge'
```

Comment:
Can't this be done in `next.config.js` with rewrites or streaming doesn't work doing it that way?

Reply:
Sounds good! Both should be technically equal at some point 👍